certified-connectors/iMIS/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4928 @@
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/ProblemDetails"
+            }
+          },
+          "500": {
+            "description": "Server Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AddressListDTO"
+            }
+          }
+        },
+        "x-ms-visibility": "important"
+      }
+    },
+    "/api/address/{contactId}/{addressPurpose}": {
+      "get": {
+        "tags": [
+          "Address"
+        ],
+        "summary": "Get an address for a contact",
+        "description": "Retrieves an address for a contact.",
+        "operationId": "AddressGet",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "path",
+            "name": "contactId",
+            "description": "The iMIS ID to get the address for",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "iMIS ID",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single"
+          },
+          {
+            "in": "path",
+            "name": "addressPurpose",
+            "description": "Must be a well-defined address purpose name in iMIS",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Address Purpose",
+            "x-ms-visibility": "important",
+            "x-ms-url-encoding": "single",
+            "x-ms-dynamic-values": {
+              "operationId": "AddressMetadataPurposeList"
+            }
+          }
+        ],
+        "responses": {
+          "400": {
+            "description": "Bad Request",
+            "schema": {
+              "$ref": "#/definitions/ProblemDetails"
+            }
+          },
+          "401": {
+            "description": "Unauthorized",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "404": {
+            "description": "Not Found",
+            "schema": {
+              "$ref": "#/definitions/ProblemDetails"
+            }
+          },
+          "500": {
+            "description": "Server Error",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/AddressDTO"
+            }
+          }
+        }
+      }
+    },
+    "/_metadata/address/purposes": {
+      "get": {
+        "tags": [
+          "AddressMetadata"
+        ],
+        "summary": "Get address purpose list",
+        "operationId": "AddressMetadataPurposeList",

```

Comment:
@csijake can you please define a description for this action? please make sure that all operations have descriptions.